patchwork/models.py

```diff
@@ -729,6 +736,15 @@ class Meta:
         ]
 
 
+class PatchReviewIntention(models.Model):
+    patch = models.ForeignKey(Patch, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    last_time_marked_for_review = models.DateTimeField(default=tz_utils.now)
```

Comment:
Can we rename this field to simply `last_updated`. I also wonder if we could/should track removals from this set? If we do, I think we likely only need to track the most recent request.

```suggestion
    last_updated = models.DateTimeField(auto_now=True)
    removed = models.BooleanField(default=False)
    removed_reason = models.CharField(max_length=50)
```